src/bot.rs

```diff
@@ -85,17 +88,17 @@ lazy_static! {
 
 pub const VOTE_BASE_COST:u16 = 40;
 #[cfg(not(feature = "debug"))]
-pub const MOTIONS_CHANNEL:u64 = 609093491150028800; //bureaucracy channel
+pub const MOTIONS_CHANNEL:u64 = 983019887024807976; //bureaucracy channel
```

Comment:
MOTIONS_CHANNEL has to be right for prod, either add another #[cfg] or reimplement with lazy_static and reading from env

Reply:
Whoops! Fixed in https://github.com/consortium-chat/plutocradroid/pull/5/commits/5b7d1aa46aa68f5139ae9bed6833c8b7ad4dafaf